StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/AccountPickerRowView.swift

```diff
@@ -26,8 +26,8 @@ final class AccountPickerRowView: UIView {
                     transform: nil
                 )
                 layer.shadowColor = UIColor.black.cgColor
-                layer.shadowRadius = 0.5
-                layer.shadowOpacity = 0.17
+                layer.shadowRadius = 1.5 / UIScreen.main.nativeScale
```

Comment:
this is an intentional somewhat unrelated change but is still pictured in the test plan (the account rows)